### DISCUSSION TITLE
Installation issue

### DISCUSSION THREAD

**Discussion by taveren** [microsoft/vcpkg#27730] [Q&A] [answered]
I can't install vcpkg on ubuntu 20 or 22 (Vsphere VM's)

__________________________________________________________________________________________________________________________________________
root@helpdesk:~# ./vcpkg/bootstrap-vcpkg.sh
Downloading vcpkg-glibc...
Telemetry
---------
vcpkg collects usage data in order to help us improve your experience.
The data collected by Microsoft is anonymous.
You can opt-out of telemetry by re-running the bootstrap-vcpkg script with -disableMetrics,
passing --disable-metrics to vcpkg on the command line,
or by setting the VCPKG_DISABLE_METRICS environment variable.

Read more about vcpkg telemetry at docs/about/privacy.md
__________________________________________________________________________________________________________________________________________

Downloading vcpkg-glibc is a problem. Any ideas on what side to dig please.

**Comment by AugP** [comment]
Hi, to clarify, is the bootstrapper hanging on that "Downloading vcpkg-glibc" step? How long was it taking?

**Reply by taveren** [reply]
> Hi, to clarify, is the bootstrapper hanging on that "Downloading vcpkg-glibc" step? How long was it taking?

Hi! It takes not more than 5 seconds. 


> > Downloading vcpkg-glibc is a problem. Any ideas on what side to dig please.
> 
> Can you clarify what the problem is? The output you showed indicated that the download was successful.

Problem is that it returns to shell after that. So script's not working.

**Reply by BillyONeal** [reply]
Returning to the shell after that is what it's supposed to do...

**Reply by Neumann-A** [reply]
> Problem is that it returns to shell after that. So script's not working.
 
you mean like:
```
neumann@Iluinrandir:/mnt/e/vcpkg_folders/x_stuff$ ./bootstrap-vcpkg.sh
Downloading vcpkg-glibc...
neumann@Iluinrandir:/mnt/e/vcpkg_folders/x_stuff$
```
That is intended. Do a `ls | grep vcpkg` and see if it shows a vcpkg binary

**Reply by taveren** [reply]
> > Problem is that it returns to shell after that. So script's not working.
> 
> you mean like:
> 
> ```
> neumann@Iluinrandir:/mnt/e/vcpkg_folders/x_stuff$ ./bootstrap-vcpkg.sh
> Downloading vcpkg-glibc...
> neumann@Iluinrandir:/mnt/e/vcpkg_folders/x_stuff$
> ```
> 
> That is intended. Do a `ls | grep vcpkg` and see if it shows a vcpkg binary

Thank you! It's my fault. It really works as it should. I expected another console output according to instruction i'm folowing.

**Comment by BillyONeal** [comment, marked as answer]
>Downloading vcpkg-glibc is a problem. Any ideas on what side to dig please.

Can you clarify what the problem is? The output you showed indicated that the download was successful.
